inc/ticket.class.php

```diff
@@ -5238,11 +5238,16 @@ static function showListForItem(CommonDBTM $item) {
             // you can only see your tickets
             if (!Session::haveRight(self::$rightname, self::READALL)) {
                $restrict .= " AND (`glpi_tickets`.`users_id_recipient` = '".Session::getLoginUserID()."'
-                                   OR (`glpi_tickets_users`.`tickets_id` = '".$item->getID()."'
+                                   OR (`glpi_tickets_users`.`tickets_id` = `glpi_tickets`.`id`
                                        AND `glpi_tickets_users`.`users_id`
                                             = '".Session::getLoginUserID()."')
                                    OR `glpi_groups_tickets`.`groups_id` IN (".implode(",",$_SESSION['glpigroups'])."))";
             }
+
+            if (Session::haveRight(self::$rightname, self::ASSIGN)) {
```

Comment:
No, the right to see new tickets is included in Ticket::READALL

Recall for other rights:
* Ticket::READASSIGN: See ticket assigned to me or my groups
* Ticket::READMY: See my own tickets (Created by me or where i am in requester/observer user of tickets)
* Ticket::READMY: See my groups tickets ( where my group(s) are in  requester/observer groups of tickets)
* **Ticket::ASSIGN**: Permit to assign (the action) tickets i can view (defined by other rights)

See https://github.com/glpi-project/glpi/blob/9.1.2/inc/ticket.class.php#L255-L277 for detail